src/Symfony/Component/Yaml/Dumper.php

```diff
@@ -43,16 +43,17 @@ public function setIndentation($num)
      * @param int   $indent                 The level of indentation (used internally)
      * @param bool  $exceptionOnInvalidType true if an exception must be thrown on invalid types (a PHP resource or object), false otherwise
      * @param bool  $objectSupport          true if object support is enabled, false otherwise
+     * @param bool  $dateTimeSupport        true if DateTime objects must be dumped as YAML timestamps, false if DateTime objects are not supported
```

Comment:
In fact I think it should not have an option for dumping because it doesn't break compatibility in that way.